README.md

```diff
@@ -1,3 +1,6 @@
+<p align="center"><img width=30% src="https://github.com/amrllkmn/healthyhome/blob/gh-pages/logo.png"></p>
```

Comment:
perhaps you could place the logo in a folder like `docs/images` in this repository and link it with simply `docs/images/logo.png`. That should work.

Reply:
You could also use a `svg` image now that I think of it.

Reply:
Oh, I actually thought about SVGs. But mainly because the image quality is maintained regardless of the scaling. Could you explain more about how it would help with future modifications? In this case, then it would be worthwhile pushing a separate logo into this branch rather than using the ```gh-pages``` branch.

Reply:
Yeah, I was referring to pushing it to the main branch under some folder like `docs/images` that you would need to create I guess. For future modifications I meant to the logo itself, just in case we lose the original files and we want to make some slight changes, we'd have some vectorial version of it.

Reply:
I understand. I'll make the necessary changes. Again, these are all new commits right?

Reply:
No reason to. You can just amend the Add logo commit, it's all part of that after all

Reply:
I agree with Luca - having a folder in the main branch could be helpful for the future.